src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml

```diff
@@ -129,10 +129,8 @@
                         else
                         {
                             <p class="govuk-body">
-                                Requests expire after @Model.Dashboard.RemovedAfterExpiryMonths months and are no longer seen by training providers.
-                            </p>
-                            <p class="govuk-body">
-                                When your requests have expired, you'll see them here
+                                Requests expire after @Model.Dashboard.RemovedAfterExpiryMonths months and are no longer seen by training providers. 
```

Comment:
In the story it is ... training providers. When you requests....

Reply:
According to AC 6, the expired tab should look like this when thre are no expired requests 
<img width="584" alt="image" src="https://github.com/user-attachments/assets/d27003f9-508d-41ac-84a9-a3c4e6e32e71">

Reply:
Yes so I was questioning the carriage return introduced from removed the p tag